commands/scanpullrequest.go

```diff
@@ -274,13 +290,20 @@ func runInstallIfNeeded(project *utils.Project, workDir string, failOnInstallati
 			err = restoreErr
 		}
 	}()
-	log.Info("Executing", "'"+project.InstallCommandName+"'", project.InstallCommandArgs, "at", workDir)
-	//#nosec G204 -- False positive - the subprocess only run after the user's approval.
-	if err = exec.Command(project.InstallCommandName, project.InstallCommandArgs...).Run(); err != nil {
-		if failOnInstallationErrors {
-			return err
+	log.Info("Executing", "'"+scanSetup.InstallCommandName+"'", scanSetup.InstallCommandArgs, "at", workDir)
+	var output []byte
+	if scanSetup.DepsResolutionRepo != "" {
+		if _, exists := utils.MapTechToResolvingFunc[scanSetup.InstallCommandName]; !exists {
```

Comment:
I believe that it is not possible, as the entire command is received  as `InstallCommand` from frogbot-config.yml and parsed as a pair of strings `InstallCommandName` and `InstallCommandArgs`. 

Basically `InstallCommandName` is the first argument in the string and 'InstallCommandArgs' are the rest. So I think it can only fail if the full install command is invalid (e.g. `install npm`)